pystac/html/jinja_env.py

```diff
@@ -0,0 +1,21 @@
+from functools import lru_cache
+from itertools import islice
+
+
+@lru_cache()
+def get_jinja_env():  # type: ignore
+    try:
+        from jinja2 import Environment, PackageLoader, select_autoescape
+
+        environment = Environment(
+            loader=PackageLoader("pystac", "html"), autoescape=select_autoescape()
+        )
+
+        environment.filters["first"] = lambda x: islice(x, 1)
+        environment.filters["is_nonempty_generator"] = (
+            lambda x: next(x, None) is not None
+        )
+
+        return environment
+    except ModuleNotFoundError:
+        return None
```

Comment:
This try block could be more narrow, correct?
```suggestion
    try:
        from jinja2 import Environment, PackageLoader, select_autoescape
    except ModuleNotFoundError:
        return None
    environment = Environment(
        loader=PackageLoader("pystac", "html"), autoescape=select_autoescape()
    )

    environment.filters["first"] = lambda x: islice(x, 1)
    environment.filters["is_nonempty_generator"] = (
        lambda x: next(x, None) is not None
    )

    return environment
```

Reply:
Correct, changed in https://github.com/stac-utils/pystac/pull/743/commits/07042aac10ef257ec0d84976985ca4d0907712d0